UI/_setup.test.ts

```diff
@@ -37,6 +38,19 @@ export async function test_db_view() {
             const res = relays.get(eventID);
             return res ? res : [];
         },
+        getAllRelayRecords: async () => {
```

Comment:
This interface can return `Map<string, Set<string>>` to make both caller & callee simpler.